Use correct encodings for vimtutor & menu translations, plus updated TR translations

bitigchi opened the pull request:
To avoid any encoding issues, it is better to use only Unicode encoding for Turkish localisations; it is only available on Unicode-supporting systems anyway.

In addition to that, this pull request includes layout fixes for `--version` and `--help` screens. I also thought it would be good idea to include the macOS folder attributes file `.DS_Store` to avoid them leaking into the git stuff. This file creates itself automatically when a folder is viewed in macOS Finder.

Commit:
use only unicode

k-takata commented:
> To avoid any encoding issues, it is better to use only Unicode encoding for Turkish localisations; it is only available on Unicode-supporting systems anyway.

It might work when Vim is compiled with +iconv feature, however, if the iconv feature is disabled, it doesn't work.

Commit:
try fixing cp1254 tutor, translate newly added strings

bitigchi commented:
I have modified the `tutor.vim` file, and added an `.iso9` version of the tutor. I was not able to test this, since I could not find a way to launch `vimtutor` from the VS 2019 Command Prompt. `vimtutor.bat` is complaining that gVim is not in the path. Or if someone could guide me on how to create an installer, that would be perfect!

Also a request, if you could double check the encoding of `.tr.iso9` file that would be great. When I reopen the file after saving, Vim still shows the encoding of the file as `utf-8`, so I am this close to jump out of the window.

Commit:
fix encoding

bitigchi commented:
Okay, now I believe I got the tutor encoding right. It shows as Latin1 after a reopen, but the way characters are substituted look accurate. I am still trying to figure out the menu translations, since I get E513 every time I try to set 'fenc'. Any help is appreciated.

k-takata commented:
I think `menu_tr.iso_8859-9.vim` and `menu_tr_tr.iso_8859-9.vim` should be kept.
And, have you tried this? https://github.com/vim/vim/issues/5209#issuecomment-552427686

> I could not find a way to launch `vimtutor` from the VS 2019 Command Prompt. `vimtutor.bat` is complaining that gVim is not in the path.

If you build gvim.exe in `src` directory, executing `..\vimtutor.bat` might be enough.

k-takata commented:
> since I get E513 every time I try to set 'fenc'.

Do you have libiconv-2.dll? Does `echo has('iconv')` return 1?

k-takata commented on runtime/tutor/tutor.tr.iso9:
```diff
@@ -22,7 +22,7 @@
   ekran� tamamen doldurmas� i�in j d��mesine yeterli miktarda bas�n.
 
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
English version of tutor doesn't have a space here. Do you intentionally insert this space?

bitigchi replied:
Yes. One space at each side, thus providing an equal look, and better visual understanding of a new chapter.

k-takata commented on runtime/tutor/tutor.tr.iso9:
```diff
@@ -1,6 +1,6 @@
-===============================================================================
+================================================================================
+=     V I M   T u t o r ' a   h o �   g e l d i n i z !    --    S�r�m 1.7     =
+================================================================================
```
English version of tutor is 79 columns here. I think it's better to keep the same width here.

bitigchi replied:
I added that extra column to make it look nicer and equal on both sides. It's up to you, but if there is not a specific reason for this it would be better this way from a visual standpoint.

brammool commented:
I think we should keep the iso9 menu file, for those users that don't have iconv support.
I'll include the rest, thanks.
